service/controller/clusternamespace/resource/clusternamespaceresources/create.go

```diff
@@ -4,8 +4,8 @@ import (
 	"context"
 	"fmt"
 
-	securityv1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/security/v1alpha1"
 	"github.com/giantswarm/microerror"
+	orgcrd "github.com/giantswarm/organization-operator/api/v1alpha1"
```

Comment:
I would use an alias of `security` as that's the name of the API group for the Organization API.

Reply:
done